css/parser/src/parser/style_rule.rs

```diff
@@ -99,7 +103,18 @@ where
         let start = self.input.cur_span()?.lo;
 
         if !is!(self, EOF) && eat!(self, "!") {
-            expect!(self, "important");
+            self.input.skip_ws()?;
+
+            let is_important = match bump!(self) {
+                Token::Ident(value) => &*value.to_ascii_lowercase() == "important",
```

Comment:
hm, yep, because no `!important`, `important = true` should be only for `!important`, https://www.w3.org/TR/css-syntax-3/#!important-diagram, in theory we can change ast and output `important: true | false | "ident-token"`, i.e. for your example it will be `important: "foo"`

Reply:
If we want to allow to using parser for linting purposes it will be useful

Reply:
@kdy1 friendly ping

Reply:
I think a warning for `red!foo` would be great, but can be fixed with follow-up PRs.

Reply:
Yep, I think we can improve not valid parsing in future - after feedback, right now let's focus on parsing problems and ast